packages/pages/src/common/src/project/path.ts

```diff
@@ -0,0 +1,22 @@
+import pathLib from "path";
+
+/**
+ * Provides useful methods to operate on a specific property of {@link ProjectFilepathsConfig}.
```

Comment:
Is `ProjectFilepathsConfig` a type or interface? I can't seem to find it defined anywhere in the repo.

Reply:
I assume `ProjectFilepathsConfig ` got rename to `ProjectFilepaths` interface and this comment was meant to point to that. I can update

Reply:
defined here https://github.com/yext/pages/blob/e58a8ec42258b56688b6ce39373f0d22dbe60c4c/packages/pages/src/common/src/project/structure.ts#L68